README.md

```diff
@@ -40,3 +40,29 @@ with parallel_backend('spark', n_jobs=3):
 
 print(scores)
 ```
+
+## Limitations
+
+`joblibspark` does not generally support run model inference and feature engineering in parallel.
+For example:
+
+```python
+from sklearn.feature_extraction import FeatureHasher
+h = FeatureHasher(n_features=10)
+with parallel_backend('spark', n_jobs=3):
+    # This won't run parallelly on spark, it will still run locally.
+    h.transform(...)
+
+from sklearn import linear_model
+regr = linear_model.LinearRegression()
+regr.fit(X_train, y_train)
+
+with parallel_backend('spark', n_jobs=3):
+    # This won't run parallelly on spark, it will still run locally.
+    regr.predict(diabetes_X_test)
+```
+
+But note that, for some algorithm, such as `sklearn.ensemble.RandomForestClassifier`, there's `n_jobs`
+parameter, that means the algorithm may support model training/inference in parallel, and its implementation
+may bind the backend to built-in backends, so the spark backend may still not work. User need check
```

Comment:
Why not say definitively whether it works or not so users feel confident? 

In order to convince users to try this library, should we include a common list of methods that we know work as well? I'm worried users will get scared away when they see so many limitations and no guarantee on what will work.

Reply:
Updated. I refine the description.
Currently, besides model selection, I only find `sklearn.ensemble.RandomForestClassifier` support the `n_jobs` parameter